docs/RELEASE-NOTES.rst

```diff
@@ -7,6 +7,26 @@ Release Notes for BIG-IP Controller for Cloud Foundry
 Added Functionality
 ```````````````````
 
+v1.1.0
+------
+
+Added Functionality
+```````````````````
+* L7 (HTTP) per route configuration through Route Services
+* Ability to run as a Service Broker to enable per route configuration
```

Comment:
Combine these two bullets: L7 (HTTP) per route configuration through Route Services when controller is run as a service broker.